### PR TITLE
[12.0][ADD] add new addon attribute_set_searchable

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
Thank you @Cedric-Pigeon  LGTM (code review only)

File: attribute_set_searchable/models/attribute_attribute.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                if not custom_filter:
+                    self._create_custom_filter()
+                else:
+                    self._update_custom_filter(custom_filter)
+            elif custom_filter:
+                custom_filter.unlink()
+        return res
+
+    @api.model
+    def create(self, vals):
```
Comment (lmignon) [inline attribute_set_searchable/models/attribute_attribute.py]: @Cedric-Pigeon With this implementation you breaks the possibility for batch creation... https://github.com/odoo/odoo/blob/12.0/odoo/models.py#L3490

We should pay attention to try to always support batch processing in all the methods.

**Review by sebastienbeau — APPROVED** [review]
Code review LGTM

**Pull request merged**